tests/lang-rust.yaml

```diff
@@ -2,24 +2,44 @@
   command: [rustc --version]
```

Comment:
Here is a working patch with some fixed tests:
```git
diff --git a/tests/lang-rust.yaml b/tests/lang-rust.yaml
index e846162..596e224 100644
--- a/tests/lang-rust.yaml
+++ b/tests/lang-rust.yaml
@@ -2,44 +2,39 @@
   command: [rustc --version]
   entrypoint: [bash, -i, -c]
   assert:
-    - status == 0
-    - stdout.indexOf("rustc") != -1
+  - status == 0
+  - stdout.indexOf("rustc 1.58") != -1
 - desc: it should have properly configured rustup
   command: [rustup show]
   entrypoint: [bash, -i, -c]
   assert:
-    - status == 0
-    - stdout.indexOf("/home/gitpod/.rustup") != -1
-    - stdout.indexOf("x86_64-unknown-linux-gnu") != -1
-    - stdout.indexOf("x86_64-unknown-linux-musl") == -1
+  - status == 0
+  - stdout.indexOf("/home/gitpod/.rustup") != -1
+  - stdout.indexOf("x86_64-unknown-linux-gnu") != -1
+  - stdout.indexOf("x86_64-unknown-linux-musl") == -1
 - desc: it should have cargo
   command: [cargo --version]
   entrypoint: [bash, -i, -c]
   assert:
-    - status == 0
-# - desc: it should have matching rustc and cargo versions
-#   command: [cargo --version; rustc --version]
-#   entrypoint: [bash, -i, -c]
-#   assert:
-#   - status == 0
-#   - stdout.split("\n")[0].split(" ")[1] == stdout.split("\n")[1].split(" ")[1]
+  - status == 0
+  - stdout.indexOf("cargo 1.58") != -1
 
 ## The below doesn't work in tests env but works in actual image shell, had to give up
-# - desc: it should have watch subcommand for cargo
-#   entrypoint: [bash, -i, -c]
-#   command: [cargo watch --help]
-#   assert:
-#     - status == 0
-#     - stdout.indexOf("cargo-watch") != -1
-# - desc: it should have one of edit(add, rm, upgrade, set-version) subcommand(s) for cargo
-#   entrypoint: [bash, -i, -c]
-#   command: [cargo, add, --help]
-#   assert:
-#     - status == 0
-#     - stdout.indexOf("cargo-add") != -1
-# - desc: it should have workspaces subcommand for cargo
-#   entrypoint: [bash, -i, -c]
-#   command: [cargo, workspaces, --help]
-#   assert:
-#     - status == 0
-#     - stdout.indexOf("cargo-workspaces") != -1
+- desc: it should have watch subcommand for cargo
+  entrypoint: [bash, -i, -c]
+  command: [cargo watch --help]
+  assert:
+  - status == 0
+  - stdout.indexOf("cargo-watch") != -1
+- desc: it should have one of edit(add, rm, upgrade, set-version) subcommand(s) for cargo
+  entrypoint: [bash, -i, -c]
+  command: [cargo add --help]
+  assert:
+  - status == 0
+  - stdout.indexOf("cargo-add") != -1
+- desc: it should have workspaces subcommand for cargo
+  entrypoint: [bash, -i, -c]
+  command: [cargo workspaces --help]
+  assert:
+  - status == 0
+  - stdout.indexOf("cargo-workspaces") != -1
```

Reply:
Please fix the alignments and spacing.

Reply:
Okay so I was able to make those three tests pass by masking the `CARGO_HOME` command which points to `/workspace/.cargo`. `cargo` is not able to locate the subcommand binaries when that's set for **dazzle test env**, surprisingly it isn't an issue when I run the image with `docker run` and try something like `cargo add --help`, I'm not sure why this happens and how to debug it for dazzle tests. It might be because that `/workspace` dir is non-existent in the resulting docker images/containers but oddly it doesn't affect `docker run localhost:5000/dazzle:full`

Reply:
Also, it's very difficult to go and manually fix all the indentations as VSCODE auto indents the lines, but guess I'll do it anyways 😅 
If this is very important, I think we should disable auto indentation for this repo by using `.vscode/settings.json` to make the contribution flow easier.

Reply:
Sorry, I can't fix the indents from VSCODE on Gitpod 😓